frontend/src/services/StorageService.ts

```diff
@@ -175,7 +175,19 @@ async function uploadImage(
 }
 
 async function uploadLogo(rawFile: File): Promise<string> {
-  return await uploadImage(rawFile, "logo", EnvConfig.contentBucket);
```

Comment:
Thoughts on having an optional parameter instead of having some duplicate code?

Reply:
I will refactor this part of the code base.